index.html

```diff
@@ -51,7 +63,8 @@ <h1>My Recent Works</h1>
     </div>
     <div class="work-section">
         <div class="story">
-            <img src="images/Img Placeholder.png" alt="portfolio-image">
+            <img src="images/Img Placeholder (2).png" class="image1" alt="portfolio-image">
+            <img src="images/Img Placeholder.png" class="image2" alt="portfolio-image">
             <div class="description-block">
                 <h1>Multi-Post Stories</h1>
                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
- [ ] Great job aligning the items to the left, but as part of the requirements you should only have one full name input instead of first name & last name 

![image](https://user-images.githubusercontent.com/89282221/215265953-511881b7-938f-4e20-8941-9e1c962f1086.png)

Reply:
figma design clearly explains I should use first name and last name for the desktop version

Reply:
- [ ] @Unleashedicon yes you are correct the Figma design shows two inputs, but in the project [Requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) it was mentioned that the form should have one input as full name instead of two, so kindly fix it ^^


![image](https://user-images.githubusercontent.com/89282221/215267158-382d4355-0e0b-4fae-b711-1234c2671d5c.png)

Reply:
now I see thanks for the reply